lib/swagger-module.ts

```diff
@@ -67,7 +68,9 @@ export class SwaggerModule {
     swaggerInitJS: string,
     yamlDocument: string,
     jsonDocument: string,
-    html: string
+    html: string,
+    jsonDocumentUrl: string,
+    yamlDocumentUrl: string
```

Comment:
nit(refactor): as this method signature is getting larger, can we - instead of adding 2 new arguments, introduce an options object instead?

```
private static serveDocuments(
    finalPath: string,
    urlLastSubdirectory: string,
    httpAdapter: HttpServer,
    swaggerInitJS: string,
    options: {
      html: string,
      yamlDocument: string,
      jsonDocument: string,
      jsonDocumentUrl: string,
      yamlDocumentUrl: string
   }
```

Reply:
Hi @kamilmysliwiec , should I do this refactor in this PR or in a new one?

Reply:
If you could apply that suggestion to this PR that would be ideal!